actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/validator/user/JsonSchemaValidator.java

```diff
@@ -1,40 +1,48 @@
 package org.sunbird.validator.user;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.everit.json.schema.Schema;
 import org.everit.json.schema.ValidationException;
 import org.everit.json.schema.loader.SchemaLoader;
 import org.json.JSONObject;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.JsonKey;
 import org.sunbird.common.models.util.LoggerUtil;
 import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.learner.util.DataCacheHandler;
+import org.sunbird.user.profile.ProfileUtil;
 
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Scanner;
 
+import static org.sunbird.common.request.orgvalidator.BaseOrgRequestValidator.ERROR_CODE;
+
 public class JsonSchemaValidator {
 
     private static LoggerUtil logger = new LoggerUtil(UserRequestValidator.class);
-
     private static Map<String, String> schemas = new HashMap<>();
-    private static final String SCHEMA_PATH = ProjectUtil.getConfigValue("schema_path");
-
-    public static void loadSchemas(){
-        File[] files = new File(SCHEMA_PATH).listFiles();
-
-        for (File file : files) {
-            logger.info(null, String.format("file :- "+file.getName()));
 
-            if (file.isFile()) {
-                schemas.put(file.getName(), read(SCHEMA_PATH + file.getName()));
+    public static void loadSchemas() {
 
+        try {
+            Map<String, String> configSettingMap = DataCacheHandler.getConfigSettings();
+            String schemaConfig = configSettingMap.get(JsonKey.EXTENDED_PROFILE_SCHEMA_CONFIG);
```

Comment:
no need to have variable configSettingMap... just use DataCacheHandler.getConfigSettings().get(key);